### PR TITLE
Clean up EventUtilsTest

### PR DESCRIPTION
**Pull request by mureinik**
Clean up the `EventUtilsTest` class in order to make it easier to read and maintain,

### CODE REVIEW

**Comment by mureinik** [comment]
The CI failure is unrelated to this PR, and also occurs on master. PR #302 should fix those errors, and once it's merged, I'll rebase this PR on top of it.

File: src/test/java/org/apache/commons/lang3/event/EventUtilsTest.java [hunk 1]
```diff
@@ -135,9 +129,9 @@ public void testBindFilteredEventsToMethod() {
         final EventCounter counter = new EventCounter();
         EventUtils.bindEventsToMethod(counter, "eventOccurred", src, MultipleEventListener.class, "event1");
         assertEquals(0, counter.getCount());
-        src.listeners.fire().event1(new PropertyChangeEvent(new Date(), "Day", Integer.valueOf(0), Integer.valueOf(1)));
+        src.listeners.fire().event1(new PropertyChangeEvent(new Date(), "Day", 0, 1));
```
Comment (britter) [inline src/test/java/org/apache/commons/lang3/event/EventUtilsTest.java]: We prefer explicit boxing of primitve values.

**Comment by mureinik** [comment]
Changes since previous push:

1. Removed the patch regarding autoboxing, as per @britter's review
2. Rebased in order to consume the CI fixes merged to master

**Comment by britter** [comment]
Thank you!

Don't forget to have a look at CONTRIBUTING.md. If you intend to contribute on a regular basis, it would be great if you could sign an Indivicual Contributor License Agreement.

**Pull request merged**